ui/.env

```diff
@@ -1,3 +1,6 @@
 KEYCLOAK_CLIENT_ID="dreams"
 KEYCLOAK_REALM=plato
-DATABASE_URL='postgres://postgres:postgres@localhost:35432/cobudget'
\ No newline at end of file
+DATABASE_URL='postgres://postgres:postgres@localhost:35432/cobudget'
+
+#JWT_KEY
+JWT_KEY=&^13YG*&546TR*TYGE78*&@&5!)O@_I)12
```

Comment:
Note to self to add this to vercel (different vars for prod and staging probably) before deploying

Reply:
we're using `COOKIE_SECRET` instead